docs/installation/production/installation.rst

```diff
@@ -9,13 +9,82 @@ This article will provide an overview of how to install Tethys Portal in a produ
 1. Install Tethys Portal
 ========================
 
-Follow the default :doc:`../../installation` instructions to install Tethys Portal with the following considerations
+Follow these steps to install Tethys Portal with the following considerations
 
 * Make sure to install the correct version.
 * Assign strong passwords to the database users.
 * You must edit the :file:`portal_config.yml` file to ensure production settings (i.e. ``DEBUG: False``, etc.)
 * Optionally, Follow the :doc:`./distributed` instructions to install Docker and the components of the software suite on separate servers.
 
+a) Install with Conda
+---------------------
+
+To install the ``tethysplatform`` into a new conda environment then run the following commands::
+
+    conda create -n tethys -c tethysplatform -c conda-forge tethysplatform
+    conda activate tethys
+
+Some of the following steps are dependent on the specific distribution where ``tethysplatform`` is installed.
+
+* Install Supervisor (See `<http://supervisord.org/installing.html>`_)
+* Install NGINX (See `<https://www.nginx.com/resources/wiki/start/topics/tutorials/install/>`_)
+* Generate Tethys portal configuration:
+
+    .. code-block::
+
+        $ tethys gen portal_config
+
+        $ tethys settings --set ALLOWED_HOSTS "<ALLOWED_HOST>" --set DATABASES.default.USER <TETHYS_DB_USERNAME> --set DATABASES.default.PASSWORD <TETHYS_DB_PASSWORD> --set DATABASES.default.PORT <TETHYS_DB_PORT> --set DATABASES.default.DIR <TETHYS_DB_DIR>
```

Comment:
Can we split these settings into separate steps:

* Create portal_config.yml
    ``<code block>``
* Set allowed hosts:
    ``<code block>``
* Set database parameters
    ``<code block with all database settings>``

Reply:
To configure with a Docker or local install, they will need to run the configure command with the PGPASS environment variable:

``PGPASS=<postgres password> tethys db configure``

Lookup the variable I'm only 70% sure it is PGPASS...